packages/custom-webpack/examples/full-cycle-app/index-html.transform.ts

```diff
@@ -0,0 +1,7 @@
+export default (targetOptions, indexHtml) => {
```

Comment:
Since it's a TS example we should showcase the types usage here.  
Please import the relevant types from `@angular-builders/custom-webpack` (if needed then reexport from `@angular-devkit`).

Reply:
In my opinion, use static type for `indexHtml` is good, but the other one is unnecessary, try to import a type and use for readonly obj just make our code more complicated. So, in that case, I think write document(readme) for it is enough.

Reply:
I disagree, having a type for these options is very helpful and gives you the exact set of options you can use. Otherwise you should go back and forth to the documentation (given it's up to date).  
If it's a problem I can add the type myself.

Reply:
Haha. It's no problem to me 😄 , just discuss and raise my concerns. Moreover, you are the owner, you can ask me to do everything because I happy to contribute to your package and willing to follow your rules. So don't talk to me that way `If it's a problem I can add the type myself.`.

Reply:
And one more concern, I think `indexHtml` is more important than `targetOptions`, so it should be the first param, some cases we no need to use `targetOptions`. Is it possible to have a breaking change in the next release?